tqdm/std.py

```diff
@@ -1290,8 +1290,9 @@ def fp_write(s):
             if leave:
                 # stats for overall rate (no weighted average)
                 self.avg_time = None
-                self.display(pos=0)
-                fp_write('\n')
+                self.display(pos=pos)
```

Comment:
Except for this line, all changes in this commit do not change the output of previous the version of `tqdm` if `position` is positive. 
I am not sure why `pos=0` was used instead of `pos=pos` even when it is the position is positive. To completely preserve previous output, this line could be made
`self.display(pos=max(0,pos))`